### DISCUSSION TITLE
Docker Development server setup issues

### DISCUSSION THREAD

**Discussion by CodingPupper3033** [inventree/InvenTree#5280] [Q&A]
Hello again,

I am trying to create a development docker server of Inventree, following the steps here: https://docs.inventree.org/en/latest/start/docker_dev/

I am able to get the server to run, however, it is not importing the test data. I have tried both the quick start and the step-by-step guide multiple times. I can see a long list of "Applying item ..." during setup, which tells me that it is working, but logging in shows no default data.

I believe this article works, as I previously setup an instance with this guide, but I deleted the files and the dockers in an attempt to restart from scratch, as some of my testing messed up the server seemingly irrecoverably.

The only difference I can think of is maybe there was an update to Inventree, and maybe it broke something? My only other "hint" is that I used to be able to login with usr:"admin" pass:"inventree" and not invoke superuser, but now I need to invoke superuser. 

Anyways, any help would be appreciated!

**Comment by wolflu05** [comment]
Hello @CodingPupper3033 ,

I can only think of that you're using different databases or data is not mounted to a volume while importing testing data and running InvenTree. So you import the data to a other db than the one InvenTree uses if you run the server. So I would double check that.

Additionally, did you know about the [devcontainers](https://docs.inventree.org/en/latest/develop/devcontainer/), I really like how easy it is to set up a development environment with it and develop plugins.

**Reply by CodingPupper3033** [reply]
I haven't used em, probably because it says visual studio code (and didn't look further), and I'm an IntelliJ type of guy. I'll see if this works, thanks!

**Reply by CodingPupper3033** [reply]
I also thought it may be looking at another database, but the docker desktop only sees one running, which I would assume means that is the one it's looking at

**Reply by wolflu05** [reply]
Yep, it's vscode only. I think I read that fleet offers devcontainers, but for the main ides like pycharm it's still in development. I tried the jetbrains ides a while ago myself, but quickly switched back to vscode, because I didn't missed anything in vscode. But I definitely give them a second try in some time. Also I was using pycharm already a couple years ago for something different, and to that time the jetbrains tools where really powerful, especially the vcs integration and debugger, but now vscode offers everything I need and it's free.

But back to the question. I would suggest inspecting the db with a database viewer like dbeaver or datagrip and see if there are the records created. Also see if invoke superuser creates a user in that db.  There is also the option to use SQLite, maybe that's setup somewhere.

**Reply by CodingPupper3033** [reply]
I think IntelliJ is free to students (and people who claim they are students). I ise my school account so I don't get any issues, but I believe Intried it on a personal account and it also worked. Dunno, maybe try getting a student free plan
